apps/aecore/src/aec_peers.erl

```diff
@@ -270,178 +236,270 @@ unblock_all() ->
 %%%=============================================================================
 
 
--record(state, {peers :: gb_trees:tree(binary(), peer()),
-                blocked = gb_sets:new() :: gb_sets:set(http_uri_uri()),
-                errored = gb_sets:new() :: gb_sets:set(http_uri_uri()),
-                local_peer :: peer(),  %% for universal handling of URIs
-                next_unblock = 0 :: integer() %% Erlang timestamp in ms.
+-record(state, {peers                   :: gb_trees:tree(binary(), peer()),
+                blocked = gb_sets:new() :: gb_sets:set(peer_id()),
+                local_peer              :: peer_info(),  %% for universal handling of URIs
+                next_unblock = 0        :: integer() %% Erlang timestamp in ms.
                }).
 
 start_link() ->
     gen_server:start_link({local, ?MODULE} ,?MODULE, ok, []).
 
 init(ok) ->
-    {Scheme, Host, Port} = aeu_env:local_peer(),
-    LocalPeer = #peer{scheme = Scheme, host = Host, port = Port},
+    {_Scheme, Host, _Port} = aeu_env:local_peer(),
+    Port = aeu_env:user_config_or_env([<<"sync">>, <<"port">>], aecore, sync_port, 1234),
```

Comment:
I see this configuration related call is all over the place, and in this particular case it's even uses hardcoded (probably) wrong default port.

Isn't it better to wrap it small function like most of the other modules do?

Reply:
Yeah, this is lazy code - will try to remember to fix it...

And yes, we should decide on a standard listen port for sync!

Reply:
DONE